tests/rt/add_product.pm

```diff
@@ -14,7 +14,7 @@ use testapi;
 sub run() {
     record_soft_failure 'poo#96158 - Adding RT product to control.xml';
     assert_screen 'startshell', 90;
-    assert_script_run 'sed -i \'/./{H;$!d} ; x ; s/\s*<\/base_product>\s*<\/base_products>/<\/base_product><base_product><display_name>SUSE Linux Enterprise Real Time 15 SP3<\/display_name><name>SLE_RT<\/name><version>15\.3<\/version><register_target>sle-15-\$arch<\/register_target><archs>x86_64<\/archs><\/base_product><\/base_products>/\' control.xml';
+    assert_script_run 'sed -i \'/./{H;$!d} ; x ; s/\s*<\/base_product>\s*<\/base_products>/<\/base_product><base_product><display_name>SUSE Linux Enterprise Real Time 15 SP4<\/display_name><name>SLE_RT<\/name><version>15\.4<\/version><register_target>sle-15-\$arch<\/register_target><archs>x86_64<\/archs><\/base_product><\/base_products>/\' control.xml';
```

Comment:
LGTM as is, but if you use `VERSION` variable (`s/-SP/ SP/` and `s/-SP/\\./`) it should work also for upcoming releases.

Reply:
Rather not touch now, but i will include it in next improvements in August.